Source/Bricks/Image/ImageBrick.swift

```diff
@@ -6,10 +6,28 @@
 //  Copyright © 2016 Wayfair LLC. All rights reserved.
 //
 
+// MARK: - Custom ImageBrick ImageView
+
+public typealias ImageSet = () -> Void
+
+/// An object that allows for manipulation of the image brick's image view
+public class ImageBrickImageView: UIImageView {
```

Comment:
Could we not figure out a right place to call the delegate instead of a custom class. I fear for people overwriting ImageBrick with custom nibs...